model/src/main/scala/cromwell/pipeline/model/validator/WrappedFactoryPattern.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.pipeline.model.validator
+
+package cromwell.pipeline.model.validator
+
+import cats.Show
+import cats.data.Validated.{ Invalid, Valid }
+import cats.data.{ NonEmptyChain, Validated }
+
+trait WrappedFactoryPattern[Type, Error, Wrapper <: Wrapped[Type]] {
+  type ValidationResult[A] = Validated[NonEmptyChain[Error], A]
+  implicit def wrappedOrdering(implicit ord: Ordering[Type]): Ordering[Wrapper] = Ordering.by(_.unwrap)
+  def create(value: Type): Wrapper
+  def validate(value: Type): ValidationResult[Type]
```

Comment:
I just noticed that you changed the access modifiers of these methods from `public` instead of `protected`.
You must keep the `protected` modifier. Otherwise, anyone can create `microtype` instance bypassing our validations (like email pattern).

Reply:
done